### ISSUE TITLE
Surface Pro - intermittent pen inaccuracy when hand is on the screen 

### ISSUE THREAD

**Issue by shawnhind** [TheJoeFin/Windows10-Community#8]
https://answers.microsoft.com/en-us/surface/forum/surfpronew-surfdrivers/surface-pro-intermittent-pen-inaccuracy-when-hand/28ef8566-0003-4c68-a964-6c2562423eb0

**Comment by gregordr** [comment]
Yes this. If it's not getting fixed I'm returning my Surface. You can't release a drawing tablet that you can't draw on properly. There are suggestions like disabling pen tilt etc.

Please tell us if there is any progress on this issue.

**Comment by TheJoeFin** [comment]
@Somethingweirdhere This is a community forum. If you find any updates from Microsoft or other resources related to this issues please link them here.

**Comment by TheJoeFin** [comment]
@shawnhind have you tried installing the latest divers from here:
https://support.microsoft.com/en-ca/help/4023482/surface-download-drivers-and-firmware-for-surface 

Let me know if this solves this issue for you.
